remove-extra-feeds.php

```diff
@@ -9,6 +9,7 @@
   Author URI:  https://github.com/gocemitevski
   License:     GPL2
   License URI: https://www.gnu.org/licenses/gpl-2.0.html
+  Copyright:   Copyright (c) 2016 Keitaro AB
```

Comment:
Adding Copyright in that way will probably result with an invalid plugin header. https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields

There is an alternative way to add the Copyright value by specifying a PHPDoc DocBlock. For example:

```
<?php
/**
 * Plugin Name
 *
 * @package     PluginPackage
 * @author      Your Name
 * @copyright   2019 Your Name or Company Name
 * @license     GPL-2.0-or-later
 *
 * @wordpress-plugin
 * Plugin Name: Plugin Name
 * Plugin URI:  https://example.com/plugin-name
 * Description: Description of the plugin.
 * Version:     1.0.0
 * Author:      Your Name
 * Author URI:  https://example.com
 * Text Domain: plugin-slug
 * License:     GPL v2 or later
 * License URI: http://www.gnu.org/licenses/gpl-2.0.txt
 */
```